projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -82,6 +82,28 @@ frontend-data-pipelines-release:
       - projects/frontend/data-pipelines/gui/package-lock.json
     expire_in: 1 week
 
+frontend_publish_ui_image:
+  image: docker:19.03.15
+  variables:
+    DOCKER_DRIVER: overlay2
+    DOCKER_TLS_CERTDIR: ''
+    DOCKER_HOST: tcp://localhost:2375
+  services:
+    - docker:19.03.15-dind
+  stage: release
+  before_script:
+    - cd projects/frontend/
+  script:
+    - docker login --username "${VDK_DOCKER_REGISTRY_USERNAME}" --password "${VDK_DOCKER_REGISTRY_PASSWORD}" "${VDK_DOCKER_REGISTRY_URL}"
+    - ./cicd/publish_image_dockerhub.sh vdk-operations-ui ./data-pipelines/gui $CI_PIPELINE_ID
+  rules:
```

Comment:
How I always test these is by changing the rule in a commit. then when it runs successfully I revert that change. 

Can you do that please.